docs/content/development/extensions-core/lookups-cached-global.md

```diff
@@ -195,12 +195,13 @@ The `namespaceParseSpec` can be one of a number of values. Each of the examples
 Only ONE file which matches the search will be used. For most implementations, the discriminator for choosing the URIs is by whichever one reports the most recent timestamp for its modification time.
 
 ### csv lookupParseSpec
-
 |Parameter|Description|Required|Default|
 |---------|-----------|--------|-------|
-|`columns`|The list of columns in the csv file|yes|`null`|
+|`columns`|The list of columns in the csv file|no if `hasHeaderRow` is set|`null`|
 |`keyColumn`|The name of the column containing the key|no|The first column|
 |`valueColumn`|The name of the column containing the value|no|The second column|
+|`hasHeaderRow`|A flag to indicate that column information can be extracted from the input files' header row|no|false|
+|`skipHeaderRows`|Number of header rows to be skipped|no|0|
```

Comment:
Just reading this doc, it's not clear how `hasHeaderRow` interacts with `skipHeaderRows`: if rows first skipped, and then `skipHeaderRows+1`-th row is used as the header row, or the very first row is used as the header row, and then `skipHeaderRows` rows are skipped.